python/lsst/sdm/schemas/dp1.yaml

```diff
@@ -6393,8 +6463,31 @@ tables:
   - name: patch
     description: Skymap patch ID
     datatype: long
+  constraints:
+  - name: ForcedSourceOnDiaobject_DiaObject
+    description: Link a ForcedSourceOnDiaObject to its associated DiaObject.
+    '@type': ForeignKey
+    columns:
+    - '#ForcedSourceOnDiaObject.diaObjectId'
+    referencedColumns:
+    - '#DiaObject.diaObjectId'
+  - name: ForcedSourceOnDiaObject_Object
+    description: Link a ForcedSourceOnDiaObject to its associated parent Object.
+    '@type': ForeignKey
+    columns:
+    - '#ForcedSourceOnDiaObject.parentObjectId'
+    referencedColumns:
+    - '#Object.objectId'
```

Comment:
I know this is confusing but "parent" in `parentObjectId` is a deblending concept and references `ForcedSourceOnDiaObject.diaObjectId`, not the Object table. And then because we've selected only "primary" objects (i.e. not deblended parents), this column is actually zero for every row in DP1.

Reply:
I think that would be the least-confusing option.

Reply:
I removed this FK constraint.